<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registerUser</class>
 <widget class="QDialog" name="registerUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="LastNameLabel">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
       </property>
       <property name="text">
        <string>Last Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lastNameEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="FirstNameLabel">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
       </property>
       <property name="text">
        <string>First name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="firstNameEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="usernameLabel">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
       </property>
       <property name="text">
        <string>Username:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="usernameEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="password1Label">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
       </property>
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="password1Edit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="password2Label">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
       </property>
       <property name="text">
        <string>Repeat password:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="password2Edit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="roleLabel">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
       </property>
       <property name="text">
        <string>Role:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="adminRadioButton">
         <property name="text">
          <string>Admin</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="professorRadioButton">
         <property name="text">
          <string>Profesor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="studentRadioButton">
         <property name="text">
          <string>Student</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="studyGroupLabel">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
       </property>
       <property name="text">
        <string>Study group:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="studyGroupEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="studyYearLabel">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
       </property>
       <property name="text">
        <string>Study year:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="studyYearEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item alignment="Qt::AlignLeft">
      <widget class="QPushButton" name="backButton">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QPushButton" name="confirmButton">
       <property name="text">
        <string>Confirm</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>backButton</sender>
   <signal>clicked()</signal>
   <receiver>registerUser</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
